src/test/kotlin/kage/crypto/scrypt/ScryptRecipientTest.kt

```diff
@@ -0,0 +1,23 @@
+package kage.crypto.scrypt
+
+import kage.utils.decodeBase64
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class ScryptRecipientTest {
+    @Test
+    fun testWrap() {
+        val recipient = ScryptRecipient("mypass".toByteArray(), 18)
+
+        val fileKey = ByteArray(32)
+
+        val stanza = recipient.wrap(fileKey).first()
+
+        val salt = stanza.args.first().decodeBase64()
+        val workFactor = stanza.args[1].toInt()
+
+        assertEquals(ScryptRecipient.SCRYPT_SALT_SIZE, salt.size)
+        assertEquals(18, workFactor)
+        assertEquals("scrypt", stanza.type)
+    }
+}
```

Comment:
This test is also not great, but we don't have unwrap yet. I could decrypt here using `stanza.body` but I would just be repeating the implementation in the test and calling decrypt on it.

Reply:
We should also mark this as `TODO` and we can improvise it later when we have `unwrap`